### ISSUE TITLE
The remix buttons os are not labeled so it is not assertable for the talkback

### ISSUE THREAD

**Issue by discapacidad5** [JideTechnology/Remix-OS-For-PC#54]
The remix buttons os are not labeled so it is not assertable for the talkback screen reader is of utmost importance they are labeled to be used by blind people around the world

We would be giving the opportunity to millions of blind people in the world who can use the benefits of android in a pc and be independent

**Comment by Neurovert** [comment]
@jide-bill Please check this issue

**Comment by jide-bill** [comment]
@discapacidad5 Thanks for bringing this up. I will let our UX and engineer to check out the possibilities.

**Comment by discapacidad5** [comment]
I attached an image where I placed buttons that are not labeled or can not be read by the talkback screen reader http://oi65.tinypic.com/2lw38kz.jpg We would also like the next version to have the talkback pre-installed in the room

Excuse me if I do not spouse clearly but I do not know English my language is Spanish I use the translator of googles

Thanks for the support

**Comment by jide-bill** [comment]
@discapacidad5 thanks for further explanation. That is very useful to help us understand the issue. for this change will take quite amount of efforts for engineering. We will have to schedule it in Feb 2017 if we have further resource for implementing this.

**Comment by discapacidad5** [comment]
The network Latin American Tiflotechnology was very happy to hear the news that soon Remix OS can be used without problem by people with visual impairment and wish that Remix OS already bring in the Kerner talkback so that it can be activated from the first start of the PC and do not require visual help for your installation
